src/main/java/dao/ShopOperationsDaoCsvImpl.java

```diff
@@ -0,0 +1,66 @@
+package dao;
+
+import fileManagers.FileManager;
+import fileManagers.FileManagerCsvImpl;
+import fruitOperationManagers.FruitOperationManager;
+import fruitOperationManagers.FruitOperationManagerCsvImpl;
+import validators.FruitDataValidator;
+import validators.FruitDataValidatorCsvImpl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ShopOperationsDaoCsvImpl implements ShopOperationsDao {
+    private final String INPUT_FILE_NAME;
```

Comment:
такий неймінг використовується тільки для констант (static final) змінні. 
```suggestion
    private final String inputFileName;
```
спробуй ще заранити локально mvn clean package, щоб перевірити код на чекстайл

Reply:
О, блин, я просто mvn clean сделала...